po/es.po

```diff
@@ -1971,15 +1970,14 @@ msgstr ""
 msgid ""
 "  -w, --warndays WARN_DAYS      set expiration warning days to WARN_DAYS\n"
 msgstr ""
-"  -w, --warndays DÍAS_AVISO     establece el aviso de caducidad a "
-"DÍAS_AVISO\n"
+"  -w, --warndays DÍAS_AVISO     establece a DÍAS_AVISO el aviso de caducidad\n"
```

Comment:
I'd prefer to translate `expiration` as `expiración` instead of `caducidad`. In my opinion `caducidad` should be used for edibles and not for software related stuff. By the way, I think you should copy the translation from the previous `--warndays`, as it is better translated there.